### DISCUSSION TITLE
Feedback on profiles

### DISCUSSION THREAD

**Discussion by MagnusThome** [phnxdrgn536/FLSun-v400#22] [General]
Some feedback on files downloaded 18 jan 🙂

BED_MESH_PROFILE LOAD=default needs to be added in a few weeks. Currently this line does nothing but a future Klipper release will NOT load the default profile automatically anymore. All according to Klipper team.

In my own opinion 1mm thick top and bottom is a bit much for "Default" profiles heights 0.15 and lower. 
Also just wondering why no skin layer at all, is this some good trick I need to learn about?
![326148567_696234082051162_8476825576640745743_n](https://user-images.githubusercontent.com/32169384/213385876-07233962-ae94-418a-9bb8-036ec320d1f2.jpg)

"Visual" profile fine 0,1mm looks good but nozzle scrape on top layer. Combing, retraction something??
![326102823_1475990992930709_1166745262986469473_n](https://user-images.githubusercontent.com/32169384/213386044-5e01dfc5-ed67-49fc-b14b-f4c940f7328d.jpg)

**Comment by slinso** [comment]
Why does this one exist: Draft - Extra Fast 0.3mm?
I tried it today without thinking too much about it, but when it printed the 2nd layer with 400mm/s speed it tried to extrude over 50 mm³/s. Of course that did not work at all. I reduced the print speed on the fly to 250mm/s and the rest of the print went fine.

According to this review (https://www.cnckitchen.com/blog/bondtech-cht-high-flow-nozzle-reviewed) that would even not be possible with a CHT volcano nozzle. That's why I do not understand the purpose of this profile. Am I missing something?

**Reply by phnxdrgn536** [reply]
I am not seeing any issues in the current profiles. Where is the 50mm³/s coming from. All of the flow rates are set at 100% no extra also have you added your filament to the materials list in cura? A lot of the data is taken from those print settings.

**Reply by slinso** [reply]
I saw the 50mm³/s on the Klipper Dashboard and I saw underexstrusion on the print. There was no such profile setting. I just wanted to know the purpose of the profile, if it leads to underextrusion.

Side note: When I put the value (0.4, 0.3, 400) into the bondtech flow rate calculator it says: 40.26mm^3/s
If the profile works for you, are you using stock components?

**Comment by phnxdrgn536** [comment]
Can you give me the following information: material, nozzle size, and if it
was 0.4 was it the default or draft intent?

**Reply by slinso** [reply]
Intent: Draft
Profile: Extra Fast - 0.3mm
Material: PLA (Extrudr NX2)
Nozzle: 0.4 FLSun

**Comment by phnxdrgn536** [comment]
Yes I have stock nozzle… have you calibrated E-steps and pressure advance?
what is your flow and flow equalization ratio settings in cura? There are
several settings that that can cause this under extrusion… also if you have
a CHT nozzle contrary to popular opinion and beliefs I’m not sure their
“high flow” is as high or higher than the stock FLSUN nozzle… if it is as
high or higher then I would look into those previously stated parameters to
address this issue. I literally spent the first 3 weeks of owning the V400
doing nothing but calibrations and fine tuning. As a result of all of the
calibration I’ve performed I have perfect extrusion, bed adhesion, infill,
overhang, and print quality at all ranges 20-400mm/s. Also keep in mind
that every printer is different and you may have done a different
calibration or not done one I have and that profiles are subjective to
those calibration. They are designed to be a general “ballpark” to get
decent prints but even those will need fine tuning from printer to printer.
I am in the process of putting together a step by step guide of my journey
start to finish on preforming all calibrations needed for a perfectly tuned
printer. As always your feedback is greatly appreciated and helps us come
up with adjustments and solutions for everyone. 😊

**Reply by slinso** [reply]
I was just assuming that the draft profile could not work because of the high flow. But apparently I am wrong.

I have the stock nozzle, too. I was only mentioning the CHT because of the advertised higher flow rate.
Maybe I have to put some more work into my calibration. I've not done everything - still using factory settings for pressure advance and resonance compensation.

I will play around with your profiles a little bit more.
Thank you very much for your effort you put into these profiles.

**Comment by phnxdrgn536** [comment]
Stay tuned to the GitHub wiki for this project. I am drawing up a step by
step guide of adjusting, calibrating, and tuning the V400

**Comment by slinso** [comment]
When I slice using the "Draft extra fast" and open the file with Klipper it shuts down immediately. It happens with different models and everytime. The problem only exists for "Draft extra fast" for me. For example "Draft fast" is working.

Here is the gcode for an simple square (30x30x1mm) slices with Draft extra fast [GCODE](https://filetransfer.io/data-package/J4OIQMof#link)

**Comment by Moicraphy** [comment]
Hello, i'm using bimetal CHT bontech (highflow) 0.4mm as described by slinso.

**I'm also experiencing underextrusion.**

_To reproduce the High Flow around 50mm3
multiply 3 times this object in cura : https://www.thingiverse.com/thing:4864333_

_I think multiplying the objects gives time for plastic to solidify, then cura enable full speed in the 0.3 Extra Fast Draft profile._

I first tryed to add 110% extrusion factor tryed also tuning the PA on the fly but no changes were visible at full speed concerning underextrusion

2 solutions seems to work:

I reduced the speed limit on the fly and the underextrusion disapear when under 250mm/s as described by slinso

I both increased the temperature to 230°C and reduced the speed limit to 320mm/s, this worked also, but get some oozing, not dramatic but i think i need to re-tune PA and retractions to print at this temperature

**In conclusion** (test need to be done with the FLSUN noozle to confirm)
I think the Extra Draft 0.3 should be limited to 250mm/s
